gridData/core.py

```diff
@@ -573,11 +573,25 @@ def save(self, filename):
         """
         self.export(filename, file_format="pickle")
 
-    def centers(self):
+    def centers(self, origin_centered=False):
         """Returns the coordinates of the centers of all grid cells as an
-        iterator."""
+        iterator.
+
+        Parameters
+        ----------
+        origin_centered : bool
+            When is set to True the origin will be set to the lower-left corner of the grid following
+            the APBS DX format specifications.
+
+            When is set to False the origin will be set to the center of the first grid cell following
+            the official DX format specifications.
+
+            .. versionchanged:: 0.6.0
+             New *origin_centered* keyword argument
```

Comment:
de-dent, so that it sits under everything
```suggestion
       .. versionchanged:: 0.6.0
          New *origin_centered* keyword argument
```